nix/pysisyphus.nix

```diff
@@ -83,10 +83,35 @@ let
         inherit text;
         name = "pysisrc";
       };
+
+  pname = "pysisyphus";
+  version = "dev";
+  description = "Python suite for optimization of stationary points on ground- and excited states PES and determination of reaction paths";
+  homepage = "https://github.com/eljost/pysisyphus";
+
+  pkgConfig = writeTextFile {
```

Comment:
Now pysisyphus also depends on "scikit-learn". It seems to be available from Nix packages.

Reply:
Done in 16d90b4 .